packages/dso-toolkit/components/Voorbeelden/Homepage.njk

```diff
@@ -1,83 +1,211 @@
+
+<style>
+  .dso-tile-grid {
+      grid-template-columns: repeat(3, 1fr);
+  }
+  
+  @media only screen and (max-width: 480px) {
+ .dso-tile-grid {
+      grid-template-columns: repeat(2, 1fr);
+  }
+}
+</style>
 <header>
   <div class="dso-logo-bar">
-    <img src="../../images/omgevingsloket.png" alt="Omgevingsloket" class="logo" />
+    <img style="margin-top: 8px" src="../../logos/omgevingsloket.svg" alt="Omgevingsloket" class="logo"/>
     <div class="dso-tools-bar">
-      {% render '@dropdown-menu-css', dropdownButton %}
+      <div class="dso-dropdown-menu">
+    <button type="button" id="actie" aria-haspopup="menu" aria-expanded="false" class="dso-tertiary"><span>J. Janssen</span></button>
+      </div>
     </div>
   </div>
   {% render '@navigations', menu -%}
 </header>
-<main>
-  <div class="row dso-banner dso-banner-implementation-specific-image" style="background-image: url('{{ '/dummy/images/hands-on-trackpad.jpg' | path }}')">
+<main id="content">
+  <div class="row dso-banner dso-banner-implementation-specific-image" style="background-image: url('../../dummy/images/hero3.png')">
```

Comment:
De `path` pipefilter is er om in de lokale en productie omgeving alsnog kloppende URLs te genereren. Waarom heb je `| path` weggehaald?

Reply:
Path is teruggezet